packages/ui/src/models/visualization/flows/abstract-camel-visual-entity.ts

```diff
@@ -232,7 +232,7 @@ export abstract class AbstractCamelVisualEntity implements BaseVisualCamelEntity
       path: ROOT_PATH,
       entity: this,
       isGroup: true,
-      icon: NodeIconResolver.getIcon(this.type),
+      icon: NodeIconResolver.getIcon(this.type, NodeIconType.VisualEntity),
```

Comment:
We need to first merge [the RouteConfiguration PR](https://github.com/KaotoIO/kaoto/pull/1011) first, so we have all VisualEntities available to add this extra argument.